package-lock.json

```diff
@@ -1,8088 +1,8 @@
 {
   "name": "@shift72/core-template",
-  "version": "0.3.6",
-  "lockfileVersion": 2,
```

Comment:
Think you need to upgrade your `npm`!

Reply:
@PeculiarGoat What version are you using?

Reply:
@PeculiarGoat Updated now